[2023-11-26 16:44:08,447] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.0.0.149:5000 - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:08,450] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:11,671] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:11] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:11,746] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:11] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:12,414] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:12] "GET /openapi/static/images/redoc.svg HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:12,461] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:12] "GET /openapi/static/images/swagger.svg HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:12,604] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:12] "GET /openapi/static/images/rapidoc.svg HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:14,042] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:14] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:14,118] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:14] "GET /openapi/static/css/swagger-ui.css HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:14,292] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:14] "GET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:14,655] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:14] "GET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:15,259] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:15] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:44:27,846] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:44:27] "GET /pacientes HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:45:05,632] ERROR log_exception() L825  Exception on /paciente [POST] - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py L825 
Traceback (most recent call last):
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\app.py", line 97, in predict
    modelo = Model.carrega_modelo(ml_path)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\model\modelo.py", line 12, in carrega_modelo
    model = pickle.load(open(path, 'rb'))
AttributeError: Can't get attribute 'ManhattanDistance' on <module 'sklearn.metrics._dist_metrics' from 'C:\\Users\\Robson\\OneDrive\\Documentos\\PUC\\SPRINT_4\\Sistema\\MVP-PUCRJ-Modulo-4\\api\\venv\\lib\\site-packages\\sklearn\\metrics\\_dist_metrics.cp310-win_amd64.pyd'>
[2023-11-26 16:45:05,642] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:45:05] "[35m[1mPOST /paciente HTTP/1.1[0m" 500 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:45:10,591] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:45:10] "GET /pacientes HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:45:16,763] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:45:16] "GET /pacientes HTTP/1.1" 200 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:45:30,335] ERROR log_exception() L825  Exception on /paciente [POST] - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py L825 
Traceback (most recent call last):
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask\app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\app.py", line 97, in predict
    modelo = Model.carrega_modelo(ml_path)
  File "C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\model\modelo.py", line 12, in carrega_modelo
    model = pickle.load(open(path, 'rb'))
AttributeError: Can't get attribute 'ManhattanDistance' on <module 'sklearn.metrics._dist_metrics' from 'C:\\Users\\Robson\\OneDrive\\Documentos\\PUC\\SPRINT_4\\Sistema\\MVP-PUCRJ-Modulo-4\\api\\venv\\lib\\site-packages\\sklearn\\metrics\\_dist_metrics.cp310-win_amd64.pyd'>
[2023-11-26 16:45:30,348] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:45:30] "[35m[1mPOST /paciente HTTP/1.1[0m" 500 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:49:15,939] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.0.0.149:5000 - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:49:15,942] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
[2023-11-26 16:49:18,960] INFO _log() L96   127.0.0.1 - - [26/Nov/2023 16:49:18] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\Robson\OneDrive\Documentos\PUC\SPRINT_4\Sistema\MVP-PUCRJ-Modulo-4\api\venv\lib\site-packages\werkzeug\_internal.py L96  
